changelog

```diff
@@ -130,6 +130,11 @@ Version 1.13.5+dev:
        The blocking behavior can be reenabled by setting pass_allied_units=no
        in the [tunnel] tag.
  * Lua API:
+   * Upgrade to Lua 5.3.3+
+     Consult http://www.lua.org/ for a full change list and updated documenation.
+     * compatibility with Lua 5.0, 5.1 and 5.2 removed
+     * bit32 functions removed
```

Comment:
Note that bit32 as added in wesnoth 1.13.x (1.12 doesn't support bit32) so i don't think we have to leave bit32 in for wesnoth 1.14